pkg/core/native/std_test.go

```diff
@@ -170,7 +170,9 @@ func TestStdLibJSON(t *testing.T) {
 func TestStdLibEncodeDecode(t *testing.T) {
 	s := newStd()
 	original := []byte("my pretty string")
+	originalUrl := []byte("Subject=test@example.com&Issuer=https://example.com")
 	encoded64 := base64.StdEncoding.EncodeToString(original)
+	encoded64Url := "U3ViamVjdD10ZXN0QGV4YW1wbGUuY29tJklzc3Vlcj1odHRwczovL2V4YW1wbGUuY29t"
```

Comment:
Add a comment explaining where did you get these data.

Reply:
It's needed because if something will be changed, we need to preserve compatibility with C#.